embedded-graphics/src/style/text_style.rs

```diff
@@ -37,17 +44,172 @@ where
     }
 }
 
+/// Text style builder.
+///
+/// Use this builder to create styles for [`Font`]s. For convenience, the [`text_style!`] macro is
```

Comment:
`TextStyle`s are applied to `Text` and not `Font`s.
Like in the `PrimitiveStyleBuilder` docs this should also describe the purpose of the `text_style` macro. And a link to `TextStyle` is missing.